packages/cli/tsconfig.json

```diff
@@ -71,11 +71,16 @@
 
     /* Interop Constraints */
     // "isolatedModules": true,                          /* Ensure that each file can be safely transpiled without relying on other imports. */
-     "allowSyntheticDefaultImports": false,             /* Allow 'import x from y' when a module doesn't have a default export. */
-    "esModuleInterop": false,                             /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables `allowSyntheticDefaultImports` for type compatibility. */
+     "allowSyntheticDefaultImports": true,             /* Allow 'import x from y' when a module doesn't have a default export. */
```

Comment:
I was able to build and run everything with these off.  Do we still need them on for something?

Reply:
We can try again and see. I was having some issues related to it when I was working on the PR.